assets/benchmarks/shaders/CMakeLists.txt

```diff
@@ -11,6 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set(INCLUDE_FILES
+  "${PPX_DIR}/assets/benchmarks/shaders/Benchmark.hlsli"
+)
+
 generate_rules_for_shader("shader_benchmarks_compute_filter_3"
     SOURCE "${PPX_DIR}/assets/benchmarks/shaders/ComputeFilter3.hlsl"
     INCLUDES ${INCLUDE_FILES}
```

Comment:
Should the include files be added only for the shaders that actually use them, or maybe that's not worth the extra complexity?

Reply:
It's a single line, fixed the dependencies to be the correct for each shader.